CompetitionTemplate/src/competition/subsystems/drive/PoseSubsystem.java

```diff
@@ -251,21 +256,32 @@ public PoseResult getDistanceFromLeftRangerfinderToLeftWall() {
     }
     
     public double getRobotPitch() {
-        if (rioRotated.get())
-        {
+        return getRealPitch() - inherentRioPitch.get();
+    }
+    
+    public double getRobotRoll() {
+        return getRealRoll() - inherentRioRoll.get();
+    }
+    
+    private double getRealPitch() {
```

Comment:
Raw has the connotation of just returning what the sensor has, whereas this still has the "is rio rotated" aspect. I'm not super happy with the name either, though...